addons/mail/static/src/models/thread_view/thread_view.js

```diff
@@ -67,6 +67,32 @@ function factory(dependencies) {
         // Private
         //----------------------------------------------------------------------
 
+        /**
+         * @private
+         * @returns {mail.messaging}
+         */
+        _computeMessaging() {
+            return [['link', this.env.messaging]];
+        }
+
+        /**
+         * @private
+         * @returns {String[]}
+         */
+        _computeTextInputSendShortcuts() {
+            if (!this.thread) {
+                return;
+            }
+            if (
+                this.env.messaging.device.isMobile ||  // The goal here is to target small screen size
```

Comment:
Please improve the comment to explain also why we need to have this this shortcut in small screen.